apps/camera/js/controllers/controls.js

```diff
@@ -27,10 +27,32 @@ function ControlsController(app) {
   this.activity = app.activity;
   this.createView();
   this.bindEvents();
+  this.l10nGet = app.l10nGet;
+  this.updateA11yLabels();
```

Comment:
Is there any point in calling this here when we know l10n.js isn't loaded? Should we not just wait for the `'localized'` event to call `.updateA11yLabels()`?